node-hub/dora-pyorbbecksdk/dora_pyorbbecksdk/main.py

```diff
@@ -146,6 +175,29 @@ def main():
             ret, frame = cv2.imencode("." + "jpeg", color_image)
             if ret:
                 node.send_output("image", pa.array(frame), {"encoding": "jpeg"})
+
+            # Get Depth data
+            depth_frame = frames.get_depth_frame()
+            if depth_frame is None:
+                continue
+            width = depth_frame.get_width()
+            height = depth_frame.get_height()
+            scale = depth_frame.get_depth_scale()
+            depth_data = np.frombuffer(depth_frame.get_data(), dtype=np.uint16)
+            depth_data = depth_data.reshape((height, width))
+            depth_data = depth_data.astype(np.float32) * scale
+            depth_data = np.where((depth_data > MIN_DEPTH) & (depth_data < MAX_DEPTH), depth_data, 0)
+            depth_data = depth_data.astype(np.uint16)
+            depth_data = temporal_filter.process(depth_data)
+            # Send Depth data
+            storage = pa.array(depth_data.ravel())
+            node.send_output("depth_data", storage)
```

Comment:
Can we rename this `depth` instead of `depth_data`.

If you follow every comment, you should be able to get a 3D visualisation with small editing of the following dataflow: https://github.com/dora-rs/dora/blob/34f78cf38ff8660e7a0ae1e1ecda9f33527d1af7/examples/realsense/camera_rerun.yaml